tests/pax/nightly/common.libsonnet

```diff
@@ -0,0 +1,61 @@
+local common = import '../common.libsonnet';
+
+{
+  NightlyPaxTest:: common.PaxTest {
+    local config = self,
+    expPath:: '',
+    extraFlags:: '',
```

Comment:
This should ideally be a list. It'll be easier to append flags as needed without having to worry about spacing. You can combine the list of flags into a string when you substitute it with something like `std.join(" ", config.extraFlags)`